utils/proxy/proxyconfig_test.go

```diff
@@ -47,7 +47,7 @@ var (
 func (s *Suite) TestGetProxy(c *gc.C) {
 	checkProxy(c, normal, "https://perfect.crime", "https://https.proxy")
 	checkProxy(c, normal, "decemberists.com", "http://http.proxy")
-	checkProxy(c, normal, "2001:db8:85a3::8a2e:370:7334", "http://http.proxy")
+	checkProxy(c, normal, "http://[2001:db8:85a3::8a2e:370:7334]", "http://http.proxy")
```

Comment:
do you know what changed here? This feels like the sort of thing that might actually impact runtime if they changed when proxy values are returned.